src/lib/viewers/error/__tests__/PreviewErrorViewer-test.js

```diff
@@ -64,12 +64,13 @@ describe('lib/viewers/error/PreviewErrorViewer', () => {
                 const extension = testCase[0];
                 const expectedIcon = testCase[1];
 
-                const message = 'reason';
+                const err = new Error()
+                err.displayMessage = 'reason'
```

Comment:
possible ';' for style on 67 & 68

Reply:
good catch, I've been switching back and for with Python a lot...